Complete login and sign up

Willylaisa opened the pull request:
basic structure of login page

Willylaisa committed:
first commit

basic structure of login page

Willylaisa committed:
fixed link between login html and css

Willylaisa committed:
Update link to css for login.html

patrickleon710 commented on login.html:
```diff
@@ -0,0 +1,44 @@
+                        </div>
+                    </div>
+                    <div class="buttons">
+                        <a href="index.html" class="login-btn">
```
Is the idea to have a link or a button for the signing/logging in?

Willylaisa replied:
I haven't found a way of using JavaScript to make a button to open another page, but please inform me if you haave any idea

patrickleon710 replied:
You could wrap the anchor tag inside a button tag.

patrickleon710 commented on login.css:
```diff
@@ -0,0 +1,100 @@
+    white-space: nowrap;
+}
+
+@media only screen and (min-width: 768px) {
```
Shouldn't it be 
     @media screen and (min-width: 768px)   ?

Willylaisa replied:
Good point ☝️
Let me check on that

patrickleon710 reviewed (commented):
* I really like the work. You've used settings I could not remember.
Thumbs up.

Brikita commented on login.html:
```diff
@@ -0,0 +1,44 @@
+        </div> 
+        <div>
+            <form class="login-section">
+                <fieldset>
```
What about adding a legend which shows Login?

Willylaisa replied:
Great idea .. let me first practise using it😂

Brikita reviewed (commented):
This is great good work.

Willylaisa committed:
Added some javascript to the login button

Simplified the html and css as well

Brikita reviewed (approved):
I think it good for now

Willylaisa committed:
Fixed javascript to make login and sign-up better

Willylaisa committed:
deleted login.html

Index.html serves the same purpose in a much better way

patrickleon710 reviewed (commented):
I noted there is no username in the form. I thought we had agreed to have both the name and username, please advise.

Willylaisa committed:
Fixed login button  

Now it shows an error message when the user first tries to log in without signing up

Willylaisa committed:
added signup option

itsrandyM reviewed (approved):
good work on the login page.

Willylaisa committed:
Delete profile.html

Willylaisa committed:
Delete profile.css
